meson.build

```diff
@@ -1826,6 +1826,21 @@ if get_option('cplayer')
             command: [osxbundle, mpv.full_path(), '@SOURCE_ROOT@'],
         )
     endif
+
+    if not win32 and not darwin
+        if meson.can_run_host_binaries()
```

Comment:
It is unclear to me what should be the value of `X-KDE-Protocols` if this fails. But I guess it can be left as is.

Reply:
The current logic is that `X-KDE-Protocols` has everything and stuff will be selectively removed if possible. Another possible alternative would be to keep a list somewhere of some subset of `X-KDE-Protocols` that we always add by default and possibly add other ones like `smb` if possible.